samples/tfm_integration/tfm_integration.rst

```diff
@@ -96,6 +96,34 @@ For Windows-based systems, please make sure you have a copy of the utility
 available on your system path. See, for example:
 `SRecord for Windows <http://srecord.sourceforge.net/windows.html>`_
 
+Images Created by the TF-M Build
+================================
+
+The TF-M build system creates executable files:
+
+* tfm_s - the secure firmware
+* tfm_ns - a nonsecure app which is discarded in favor of the Zephyr app
+* bl2 - mcuboot, if enabled
+
+For each of these, it creates .bin, .hex, .elf, and .axf files.
+
+The TF-M build system also creates signed variants of tfm_s and tfm_ns, and a file which combines them:
+
+* tfm_s_signed
+* tfm_ns_signed
+* tfm_s_ns_signed
+
+For each of these, only .bin files are created.
+The Zephyr build system usually signs both tfm_s and the Zephyr app itself, see below.
+
+File paths for all of these files can be found as Cmake target_properties on the target 'tfm', which are preferably accessed via generator expressions:
```

Comment:
would remove preferable.
On CMake functions that supports generator expression this is for sure the best way to do it, but there could be situations where this is not possible.

So probably just something like:
```suggestion
The 'tfm' target contains properties of the form, ``<property> - <value>``, as:
*  ``<FILE_BASE_NAME>_BIN_FILE`` - ``<path>/<file_base_name>``.bin
*  ``<FILE_BASE_NAME>_HEX_FILE`` - ``<path>/<file_base_name>``.hex
where ``FILE_BASE_NAME`` is in upper case, as example ``TFM_S_SIGNED``, and ``file_base_name`` is lower case, as example: ``tfm_s_signed``.
```

Reply:
Thanks. I opted to be less verbose, but take a look.